libs/component-library/src/lib/tabs/tabs.tsx

```diff
@@ -117,22 +197,22 @@ export const FxTabs = ({
   });
 
   return (
-    <FxBox
-      flexDirection="row"
-      onLayout={(event) => {
-        containerWidth.value = event.nativeEvent.layout.width;
-      }}
-    >
-      {items.map((item, idx) => {
-        return (
-          <FixedTab
-            text={item}
-            key={idx}
-            selected={idx === selectedIdx}
-            onPress={() => selectHandler(idx)}
-          />
-        );
-      })}
+    <FxBox flexDirection="row">
+      {variant === 'auto'
+        ? tabsItems.reduce((acc, cur) => {
+            if (!acc) {
+              return cur;
+            } else {
+              return (
+                <>
+                  {acc}
+                  <FxSpacer width={24} />
```

Comment:
Should this space be able to be set by the user through some prop?

Reply:
I don't think it's necessary. The designs seemed to always indicate a spacing of 24